caret installs broken

out of the box after `npm -g i ied`:

```
$ ied install semver@^5.0.1

/usr/local/lib/node_modules/ied/node_modules/rxjs/Observer.js:5
    error: function (err) { throw err; },
                            ^
AssertionError: error status code https://registry.npmjs.org/semver/%3E=5.0.1%20%3C6.0.0: undefined
    at checkStatus (/usr/local/lib/node_modules/ied/lib/registry.js:97:20)
    at SafeSubscriber._next (/usr/local/lib/node_modules/ied/lib/registry.js:134:12)
    at SafeSubscriber.__tryOrSetError (/usr/local/lib/node_modules/ied/node_modules/rxjs/Subscriber.js:232:16)
    at SafeSubscriber.next (/usr/local/lib/node_modules/ied/node_modules/rxjs/Subscriber.js:174:27)
    at Subscriber._next (/usr/local/lib/node_modules/ied/node_modules/rxjs/Subscriber.js:125:26)
    at Subscriber.next (/usr/local/lib/node_modules/ied/node_modules/rxjs/Subscriber.js:89:18)
    at DoSubscriber._next (/usr/local/lib/node_modules/ied/node_modules/rxjs/operator/do.js:82:24)
    at DoSubscriber.Subscriber.next (/usr/local/lib/node_modules/ied/node_modules/rxjs/Subscriber.js:89:18)
    at /usr/local/lib/node_modules/ied/lib/util.js:91:14
```

maybe caused by https://github.com/tomas/needle/commit/f62e0958b8d24f6231f7611e043a08a0deddd7e7 ?
released 2 days ago in v1.1.0: https://github.com/tomas/needle/releases/tag/v1.1.0

* **Version**: ied version 2.3.1
* **Platform**: OS X El Capitan 10.11.4, Node 6.2.0
* **Subsystem**: ?

I also tested on Ubuntu on EC2 with Node 4.4.7 and had same issue.

--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     "lodash.memoize": "^4.1.0",
     "minimist": "^1.2.0",
     "mkdirp": "^0.5.1",
-    "needle": "^1.0.0",
+    "needle": "1.0.0",
     "node-gyp": "^3.4.0",
     "node-pre-gyp": "^0.6.29",
     "node-uuid": "^1.4.7",

